### PR TITLE
Merge master into ah var store

### PR DESCRIPTION
**Pull request by mcovarr**

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFHeaderLines.java [hunk 1]
```diff
@@ -90,6 +92,11 @@ public static VCFFormatHeaderLine getEquivalentFormatHeaderLine(final String inf
         addFilterLine(new VCFFilterHeaderLine(LOW_QUAL_FILTER_NAME, "Low quality"));
         addFilterLine(new VCFFilterHeaderLine(VCFConstants.PASSES_FILTERS_v4, "Site contains at least one allele that passes filters"));
 
+        addFilterLine(new VCFFilterHeaderLine(NAY_FROM_YNG, "Considered a NAY in the Yay, Nay, Grey table"));
+        addFilterLine(new VCFFilterHeaderLine(EXCESS_ALLELES,"Site has an excess of alternate alleles based on the input threshold (default is " + CommonCode.EXCESS_ALLELES_THRESHOLD + ")"));
+        addFilterLine(new VCFFilterHeaderLine(NO_HQ_GENOTYPES, "Site has no high quality variant genotypes"));
+        addFilterLine(new VCFFilterHeaderLine(EXCESS_HET_KEY, "Site has excess het value larger than the threshold"));
```
[the remark this answers is not in the document]
Comment (RoriCremer) [reply src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFHeaderLines.java:98]: note that this is for EXCESS_HET_KEY as a FILTER option, not as an INFO one though. Maybe there is someplace where they are being swapped

[the remark this answers is not in the document]
Comment (RoriCremer) [reply src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFHeaderLines.java:98]: (see line 167)

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortEngine.java [hunk 2]
```diff
@@ -0,0 +1,1232 @@
...
+        return filteredVC;
+    }
+
+    protected static VariantContext removeAnnotations(VariantContext vc) {
+
+        final VariantContextBuilder builder = new VariantContextBuilder(vc);
+        List<String> rmAnnotationList = new ArrayList<>(Arrays.asList(
+                GATKVCFConstants.STRAND_ODDS_RATIO_KEY,
+                GATKVCFConstants.AS_QUAL_BY_DEPTH_KEY,
+                GATKVCFConstants.FISHER_STRAND_KEY));
```
Comment (RoriCremer) [inline src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortEngine.java:632]: add to this list:
                GATKVCFConstants.EXCESS_HET_KEY,


(and also add to the unit test testRemoveAnnotations in  ExtractCohortRemoveAnnotationsUnitTest:
        Assert.assertFalse(expectedVC.hasAttribute(GATKVCFConstants.EXCESS_HET_KEY), "removeAnnotations did not remove ExcessHet annotation.");
)

Comment (RoriCremer) [inline src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortEngine.java:632]:
```suggestion
                GATKVCFConstants.EXCESS_HET_KEY,
                GATKVCFConstants.FISHER_STRAND_KEY));
```

**Comment by RoriCremer** [comment]
for HtsgetReader, use the master branch code only

**Comment by mcovarr** [comment]
obsolete

**Pull request closed without merging**